niworkflows/utils/tests/test_misc.py

```diff
@@ -34,8 +34,9 @@ def test_fs_license_check(stdout, rc, valid):
         assert check_valid_fs_license() is valid
 
 
-@pytest.mark.skipif(not os.getenv("FS_LICENSE"), reason="No FS license found")
-def test_fs_license_check2(monkeypatch):
+@pytest.mark.skipif(shutil.which('mri_convert') is None, reason="FreeSurfer not installed")
+@pytest.mark.xfail(not os.getenv("FS_LICENSE"), reason="No FS license found")
+def test_fs_license_check2():
```

Comment:
I think this is a more reasonable marking, since I was already fiddling with this file. Can remove it, if you disagree.